src/main/kotlin/domain/categories/add/AddCategory.kt

```diff
@@ -0,0 +1,4 @@
+package org.qudus.squad.domain.categories.add
+
+interface AddCategory {
```

Comment:
You may refactor the name to a more descriptive name.
```suggestion
interface AddNewCategory {
```